code/modules/supply/supply_packs/pack_miscellaneous.dm

```diff
@@ -0,0 +1,526 @@
+/datum/supply_packs/misc
+	name = "HEADER"
+	group = SUPPLY_MISC
+
+/datum/supply_packs/misc/mule
+	name = "MULEbot Crate"
+	contains = list(/mob/living/simple_animal/bot/mulebot)
+	cost = 250
+	containertype = /obj/structure/largecrate/mule
+	containername = "\improper MULEbot crate"
+	department_restrictions = list(DEPARTMENT_SUPPLY)
+
+/datum/supply_packs/misc/watertank
+	name = "Water Tank Crate"
+	contains = list(/obj/structure/reagent_dispensers/watertank)
+	cost = 50
+	containertype = /obj/structure/largecrate
+	containername = "water tank crate"
+
+/datum/supply_packs/misc/hightank
+	name = "High-Capacity Water Tank Crate"
+	contains = list(/obj/structure/reagent_dispensers/watertank/high)
+	cost = 12
```

Comment:
Cost in need of adjustment, common water tank is 50 credits.

Reply:
addressed in next next commit